kore/app/exec/Main.hs

```diff
@@ -370,20 +381,25 @@ main = do
     Foldable.forM_ (localOptions options) mainWithOptions
 
 mainWithOptions :: KoreExecOptions -> IO ()
-mainWithOptions execOptions@KoreExecOptions { koreLogOptions } =
-    (=<<) exitWith $ runLoggerT koreLogOptions $ case () of
-    ()
-      | Just proveOptions <- koreProveOptions execOptions ->
-        koreProve execOptions proveOptions
+mainWithOptions execOptions@KoreExecOptions { koreLogOptions } = do
+    exitCode <- runLoggerT koreLogOptions
+        $ case () of
```

Comment:
I know it was here before, but instead of the case unit trick to get guards, can't we use a function in a where clause instead?

Reply:
Done!